### DISCUSSION TITLE
ninjaaaaamuffffffiiiiiiinnnn..... where si my friday nifht funkin weekend 1 source code.... father is angry.

### DISCUSSION THREAD

**Discussion by abandonedacc2123** [FunkinCrew/Funkin#2251] [General]
.

**Comment by Johferson** [comment]
The source code of weekend 1 is at the assets folder. All of the code responsible for cutscenes, note events, and Animate Atlas characters are found in assets/scripts.

**Reply by AbnormalPoof** [reply]
Not entirely correct. That is simply the code for the level itself. They were referring to the WeekEnd 1's update source code, which has already been released.
